package.json

```diff
@@ -169,142 +157,88 @@
                 "category": "Azure Container Apps"
             }
         ],
-        "views": {
-            "azure": [
-                {
-                    "id": "containerApps",
-                    "name": "Container Apps"
-                }
-            ]
-        },
         "menus": {
-            "view/title": [
-                {
-                    "command": "containerApps.createContainerApp",
-                    "when": "view == containerApps",
-                    "group": "navigation@1"
-                },
-                {
-                    "command": "containerApps.refresh",
-                    "when": "view == containerApps",
-                    "group": "navigation@2"
-                }
-            ],
             "view/item/context": [
-                {
-                    "command": "containerApps.selectSubscriptions",
-                    "when": "view == containerApps && viewItem == azureextensionui.azureSubscription",
-                    "group": "inline"
-                },
-                {
-                    "command": "containerApps.refresh",
-                    "when": "view == containerApps && viewItem =~ /azureSubscription|managedEnvironment|containerApp/i",
-                    "group": "z@2"
-                },
                 {
                     "command": "containerApps.createManagedEnvironment",
-                    "when": "view == containerApps && viewItem =~ /azureSubscription/i",
+                    "when": "view == azureResourceGroups && viewItem =~ /azureResourceTypeGroup.*microsoft.app/managedenvironments/",
                     "group": "2@1"
                 },
                 {
                     "command": "containerApps.createContainerApp",
-                    "when": "view == containerApps && viewItem =~ /managedEnvironment/i",
```

Comment:
Thanks for bringing this one up.  This was actually meant to be a temporary fix but I had forgotten to come back to it.  I did it because of this overlap:

![image](https://user-images.githubusercontent.com/40250218/182498828-34e20fcd-aeaf-4786-bdf7-df9bb76db56a.png)

![image](https://user-images.githubusercontent.com/40250218/182498894-7ac7850d-a69a-45cc-9f09-4aec760302cc.png)

After talking with Nathan about it, I think I will end up changing the `contextValue` from `managedEnvironment` to something like `containerEnvironment` instead